pkgs/development/tools/zigmod/buildZigmodPackage.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, cacert
+, git
+, zig
+, zigmod
+}:
+
+{ name ? "${attrs.pname}-${attrs.version}"
+, src
+, fetchOutputHash
+, nativeBuildInputs ? [ ]
+, postUnpack ? ""
+, preBuild ? ""
+, meta ? { }
+, ...
+} @ attrs:
+let
+  depsDir = ".zigmod";
+
+  deps = stdenv.mkDerivation {
+    inherit src;
+
+    name = "${name}-deps";
```

Comment:
This would force us to refetch drops on every update. Is that necessary?

Reply:
Dependencies are likely to be updated with the update of the package. Or do you mean something else?

Reply:
They are likely yes but we don't need ot rebuild the FOD if really nothing changed.